src/assets/utils/formattedDate.js

```diff
@@ -0,0 +1,17 @@
+export const formattedWeekdayAndDate = (date) => {
```

Comment:
I don't think this one should be in `src/assets/utils` but rather `src/utils` - assets is usually for... well, assets like images, fonts, icons 😄